src/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { join } from 'path';
+
+import { ANDROID_MAIN_PATH } from './modules/config';
+import { readFile } from './services/file.processing';
+
+const ANDROID_MAIN_MANIFEST_PATH = `${ANDROID_MAIN_PATH}/AndroidManifest.xml`;
+const IOS_PODFILE_PATH = `./ios/Podfile`;
+
+//Resort to read file content to fetch package value rather than adding package to parse xml/podfile
+const ANDROID_PACKAGE_EXP = new RegExp(/package=\"(.+)\"/);
+const IOS_PACKAGE_EXP = new RegExp(/target\s*["'](.+)['"]\s*do/);
+
+function fetchFileLocation(filePath: string): string {
+  return join(process.cwd(), filePath);
+}
+
+function getPackageFileName(platform: string): string {
```

Comment:
should split into a `getAndroidPackageFileName` and `getIosPackageFileName`

Reply:
Just a small challenge, PR is too useful for this to prevent merging

Reply:
@yleflour  Sorry, just seeing this. Been in the cave.
Do I need to do anything on this, or can it be ignored till future updates?